package-overrides/github/angular-ui/angular-ui-router-bower@0.3.1.json

```diff
@@ -0,0 +1,11 @@
+{
+  "main": "release/angular-ui-router.js",
+  "meta": {
+    "release/angular-ui-router.js": {
```

Comment:
For jspm 0.16 compatibility it is still best to use the

```
"shim": {
  "release/angular-ui/router": {
    "deps": ["angular"]
  }
}
```

approach, at least until jspm 0.17 is out.

Also `angular` should probably be a `dependencies: { "angular": "*" }` as well?